File: 백준/Bronze/2588. 곱셈/곱셈.py
import sys
input = sys.stdin.readline

N = int(input())
M = int(input())

# N, M은 모두 세자리 수
# 1. N * M의 1의자리
# 2. N * M의 10의 자리
# 3. N * M의 100의 자리
# 4. N * M

print(N * (M % 10))
print(N * ((M % 100) // 10))
print(N * (M // 100))
print(N * M)

File: 백준/Bronze/2588. 곱셈/곱셈.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split('\n');

let [a, b] = input.map(Number);

let result = 0;
for (let i = 0; i < 3; i++) {
  let ans = a * (b % 10);
  result += ans * 10 ** i;
  console.log(ans);
  b = parseInt(b / 10);
}

console.log(result);
